### PR TITLE
Remove test env from resolve uri tests

### PR DESCRIPTION
**Pull request by nerfZael**

### CODE REVIEW

File: packages/js/client/src/__tests__/e2e/test-cases.ts [hunk 1]
```diff
@@ -5,28 +5,25 @@ export const runAsyncifyTest = async (
   client: PolywrapClient,
   wrapperUri: string
 ) => {
-  const subsequentInvokes = await client.query<{
+  const subsequentInvokes = await client.invoke<{
     subsequentInvokes: string;
   }>({
```
Comment (cbrzn) [inline packages/js/client/src/__tests__/e2e/test-cases.ts]:
```suggestion
  const subsequentInvokes = await client.invoke<string>({
```

File: packages/js/client/src/__tests__/e2e/test-cases.ts [hunk 2]
```diff
@@ -845,128 +844,58 @@ export const runSimpleStorageTest = async (
...
+  expect(getDataResult.error).toBeFalsy();
+  expect(getDataResult.data).toEqual(55);
 };
 
 export const runSimpleEnvTest = async (
   client: PolywrapClient,
   wrapperUri: string
 ) => {
-  const queryGetEnv = await client.query({
+  const queryGetEnv = await client.invoke({
```
Comment (cbrzn) [inline packages/js/client/src/__tests__/e2e/test-cases.ts]: `invokeGetEnv` instead of `queryGetEnv` maybe?

Comment (cbrzn) [reply packages/js/client/src/__tests__/e2e/test-cases.ts]: I am seeing that this is done in multiple parts, so I am not commenting on all of those but rather making a suggestion here to change it in every part `query` is mentioned :P

File: packages/test-cases/cases/wrappers/wasm-as/simple-fs-resolver/package.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "test-case-simple-fs-resolver",
+  "private": true,
+  "dependencies": {
+    "@polywrap/wasm-as": "0.1.0",
```
Comment (cbrzn) [inline packages/test-cases/cases/wrappers/wasm-as/simple-fs-resolver/package.json]:
```suggestion
    "@polywrap/wasm-as": "0.2.0",
```

File: packages/test-cases/cases/wrappers/wasm-as/simple-redirect-resolver/package.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "test-case-simple-redirect-resolver",
+  "private": true,
+  "dependencies": {
+    "@polywrap/wasm-as": "0.1.0",
```
Comment (cbrzn) [inline packages/test-cases/cases/wrappers/wasm-as/simple-redirect-resolver/package.json]:
```suggestion
    "@polywrap/wasm-as": "0.2.0",
```

File: packages/js/client/src/__tests__/e2e/test-cases.ts (hunk 2 above)
Comment (nerfZael) [reply packages/js/client/src/__tests__/e2e/test-cases.ts]: I prefer `getEnvResult`

**Pull request merged**